agents/specialized/code_generator.py

```
@@ -39,137 +40,143 @@ async def {function_name}({parameters}):
         intent_lower = intent.lower()
-        
-        if 'crud' in intent_lower:
-            return 'crud'
-        elif 'model' in intent_lower or 'schema' in intent_lower:
-            return 'data_model'
-        elif 'endpoint' in intent_lower or 'api' in intent_lower:
-            return 'api_endpoint'
+
+        if "crud" in intent_lower:
+            return "crud"
+        elif "model" in intent_lower or "schema" in intent_lower:
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)